src/molecule.py

```diff
@@ -2866,7 +2843,7 @@ def read_xyz(self, fnm, **kwargs):
         """ .xyz files can be TINKER formatted which is why we have the try/except here. """
         try:
             return self.read_xyz0(fnm, **kwargs)
-        except ActuallyArcError:
```

Comment:
The lines highlighted in red are newer, so the changes here need to be reverted, perhaps by merging the master branch into this branch.